aws-codegurureviewer-repositoryassociation/resource-role.yaml

```diff
@@ -29,6 +29,8 @@ Resources:
                 - "codeguru-reviewer:AssociateRepository"
                 - "codeguru-reviewer:DescribeRepositoryAssociation"
                 - "codeguru-reviewer:DisassociateRepository"
+                - "codeguru-reviewer:ListTagsForResource"
+                - "codeguru-reviewer:TagResource"
```

Comment:
how about removing tags ? would the tags get removed as part of disassociate?

Reply:
Tags will not be remove when disassociation. Tags will be removed when the associationId automatically deletes after in disassociated status of 90 days.